installation-bundle/src/Config/ParameterDumper.php

```diff
@@ -12,11 +12,20 @@
 
 namespace Contao\InstallationBundle\Config;
 
+use Symfony\Component\Filesystem\Exception\IOException;
 use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
 use Symfony\Component\Yaml\Yaml;
 
 class ParameterDumper
 {
+    private const CONFIG_FILES = [
```

Comment:
Since this is only used in the constructor, I would prefer a static array in the method.